blackmagic/blueprints/tile.py

```diff
@@ -221,7 +219,9 @@ def tile_fn():
         f = partial(pipeline, date=d)
         data = __workers.map(f, c)
         
-        _ = save(tx=x, ty=y, model=train(list(flatten(data)), parameters()))
+        _ = save(tx=x,
+                 ty=y,
+                 model=train(sample(numpy.array(list(flatten(data)), parameters()))))
```

Comment:
you call the sample() fn, though it doesn't do anything yet (just 'pass').  is that intentional?